src/test/java/edu/harvard/iq/dataverse/export/dataset-all-defaults.txt

```diff
@@ -14,8 +14,10 @@
     "lastUpdateTime": "2015-09-29T17:47:35Z",
     "releaseTime": "2015-09-29T17:47:35Z",
     "createTime": "2015-09-24T16:47:50Z",
-    "license": "CC0",
-    "termsOfUse": "CC0 Waiver",
+    "license": {
+          "label": "CC0-1.0",
+          "uri": "https://creativecommons.org/publicdomain/zero/1.0/"
+        },
```

Comment:
Tricky again. If I understand, this is the internal JSON export. The old version would now be considered invalid (you can't have a license and termsOfUse both filled in) so cut/paste to import it would break (as the PR is, we could add custom CC0 handling to support old exports). 

That said, in the PR, I'm not sure why we have "label" (versus "name" or "text") and for export/import purposes, just referencing the url might be the simplest way to go (to import, the license URL would have to match one of the installed options but if someone has used a different license name (as discussed above for adding 'Waiver' or internationalizing, we'd have to decide whether to ignore a name mismatch of fail the import. Sending only the URL would avoid that.)

Not sure what the best option is here - I guess I'd suggest the same `"license":"https://creativecommons.org/publicdomain/zero/1.0/"` form as in the json-ld.

Reply:
Obviously, if we end up with a backward incompatible change (which sounds likely), we should document it. Yes, this is the native JSON format and I'm sure it's widely used though I don't know how often people specify license or termsOfUse with it. @jggautier may have a better idea of this.

What I like about the old way is that specifying a license is easy and short: `license: CC0`.

I don't really love `license: URL`. How is the user the know what URL to use?

I'm also confused about "license: label, uri" that we see currently. Why is both label and uri needed? Isn't one or the other sufficient? What if they conflict (CC0 label but CC BY uri)? I guess this is the mismatch Jim is talking about above.

It all goes back to what the identifier is. In this context I like the idea of standardizing on SPDX's short identifiers like `CC-1.0` and not needing to specify a uri. You just look at https://spdx.org/licenses/ to figure out the identifier. So this would be `license: term`, pretty much the old way but using a standard list.